source/core/GenericFunctions.cpp

```diff
@@ -29,7 +29,8 @@ namespace Vireo
 ConstCStr CopyProcName(void *pSource, void *pDest, Int32 aqSize)
 {
     // If the source or dest are not aligned to aqSize bytes, return null.
-    if (((uintptr_t)pSource % aqSize != 0) || ((uintptr_t)pDest % aqSize != 0))
+    Int32 alignment = aqSize > sizeof(void*) ? sizeof(void*) : aqSize;
```

Comment:
Consider updating the comment to give a better explanation as to why we are doing this.

Reply:
Done.  Also called AQAlignment instead of computing alignment manually.